apps/web/src/server/views/inspector/components/appeal-table.component.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+
+{% macro appealTable(params) %}
+  {% set head = [] %}
+  {% set rows = [] %}
+
+  {% for column in params.columns %}
+    {% if params.headerHidden and params.headerHidden | includes(column) %}
+      {% set head = (head.push({ html: '<span class="govuk-visually-hidden">' + column + '</span>' }), head) %}
+    {% else %}
+      {% set head = (head.push({ text: column }), head) %}
+    {% endif %}
+  {% endfor %}
+
+  {% for appeal in params.appeals %}
+    {% set cells = [] %}
+    
+    {% for column in params.columns %}
+      {% set cell = {} %}
+
+      {% if column === 'Appeal age' %}
```

Comment:
Although it's not mentioned in the docs for some reason, there is a `switch` statement in Nunjucks: https://github.com/mozilla/nunjucks/pull/1034 - it might be worth using that here but it's just a suggestion.